### DISCUSSION TITLE
db.update can i use doc_id???

### DISCUSSION THREAD

**Discussion by Damien455** [msiemens/tinydb#364] [Q&A] [answered]
HI wondering if I can use doc_id = 1 when using db.update

i.e.

db.update({"Name": "Damien, "Seen": True), doc_id = 1)   

the example of your guide uses only mentions doc_id = [list]  so I could do doc_ids = [1,] but seems a little "wrong" touse a list for one item.

thanks

Damien

**Comment by eugene-eeo** [comment, marked as answer]
[Nope](https://github.com/msiemens/tinydb/blob/f9216c687c8c2d3fcfc343dcea00be101045169f/tinydb/table.py#L328), you can't use an integer in place -- using `doc_ids=[doc_id]` is fine.

**Reply by Damien455** [reply]
thanks :)
